mongo/mongotest/mgo_job_store_test.go

```diff
@@ -1,8 +1,8 @@
-package store
+package mongotest
```

Comment:
This test file should be renamed and moved to the parent directory so the test is in the same dir as the thing it's testing.
The filename should then be consistent but with `_test.go`. I'd suggest the package code being called `job_store.go` with this file being called `job_store_test.go`. The `package` in this file should then be `mongo_test` which is a special package name that allows tests to be in the same dir as the code.
```suggestion
package mongo_test
```

For an example of this from another service, look at [the api package in the dp-image-api service](https://github.com/ONSdigital/dp-image-api/blob/develop/api/image_test.go#L1).

Reply:
that's interesting to know - I've changed it now - thanks